cypress/e2e/cloud/globalHeaderOrgMenuItems.test.ts

```diff
@@ -0,0 +1,80 @@
+import {makeQuartzUseIDPEOrgID} from 'cypress/support/Utils'
+
+const globalHeaderFeatureFlags = {
+  multiOrg: true,
+  createDeleteOrgs: true,
+}
+
```

Comment:
I might call this something other than `globalHeaderFeatureFlags`, like `createOrgsFeatureFlags` so that it's distinct from the variable in the other file - otherwise folks may think later on that these variables should have the same value in both files (which they shouldn't).